float64 pose_tra_x
float64 pose_tra_y
float64 pose_tra_z
float64 looking
float64 gesture
bool result_interact
float64 box_h
float64 box_w
float64 box_x
float64 box_y
int64 id_model

